wpimath/src/test/native/cpp/controller/BangBangInputOutputTest.cpp

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#include "frc/controller/BangBangController.h"
+#include "gtest/gtest.h"
+
+class BangBangInputOutputTest : public testing::Test {
+ protected:
+  frc::BangBangController* controller;
```

Comment:
You can just make a frc::BangBangController by value instead of using new and delete. Each TEST_F() block gets its own instance of the fixture.